examples/conll2000_chunking_crf.py

```diff
@@ -0,0 +1,99 @@
+"""Train CRF and BiLSTM-CRF on CONLL2000 chunking data, similar to https://arxiv.org/pdf/1508.01991v1.pdf.
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+
+import numpy
+from collections import Counter
+from sklearn.metrics import classification_report
```

Comment:
Should we try to avoid those dependencies (`sklearn` and `nltk`)?
I agree this is very convenient and not hard to install but it could be easy to add a new dataset as in #25.